UnitedIncome/components
# [APP-4371] Auto-resize multi-line input

### Description
Monique and I discussed how to resolve the weird display issues with multi-line inputs without having to significantly re-architect the input component or create a brand new component. We decided to not to allow user resizing of the field. Instead, it auto resizes vertically as more rows are entered into the field.

### Testing Instructions
1. Go to the multiline input story
1. Enter multiple lines of text into the text field
1. The field should grow or shrink vertically with the number of text rows

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/app-4371)
